webapp/src/components/sidebar_right/index.js

```diff
@@ -4,17 +4,15 @@
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 
-import {getIssues, getInIssues, getOutIssues, getSiteURL, getTodoToast} from '../../selectors';
-import {remove, list, openAssigneeModal, openAddCard, closeAddCard, complete, bump, accept, telemetry, setRhsVisible} from '../../actions';
+import {getSiteURL, getTodoToast} from '../../selectors';
+import {remove, fetchAllIssue, openAssigneeModal, openAddCard, closeAddCard, complete, bump, accept, telemetry, setRhsVisible} from '../../actions';
 
 import SidebarRight from './sidebar_right.jsx';
 
 function mapStateToProps(state) {
     return {
-        todos: getIssues(state),
+        allIssues: state['plugins-com.mattermost.plugin-todo'].allIssues,
```

Comment:
We should use the selectors whenever possible. Reading the previous code, I think we can continue using the `getInIssues` etc. selectors to make the consumer code less complicated. The selector would call the `getAllIssues` selector and return `allIssues.in`

Reply:
I also like the convention of `myTodos` rather than just `todos`